index.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;0,700;1,400;1,700&display=swap"
+        rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+    <script defer src="./scripts.js"></script>
+
+    <title>Avengers Conference</title>
+</head>
+
+<body>
+    <header class="mobile-header">
+        <div class="icon1">
+            <img src="images/mobil_menu.png" alt="hamburger menu" onClick="menuOpen(1)" />
+        </div>
+        <div class="logo">
+            <img src="./images/logo_bogota.jpg" alt="logo" />
+        </div>
+        <ul class="menu-deployed">
+            <li><a href="./about.html " onClick="menuClose(1);">About</a></li>
+            <li><a href="#program " onClick="menuClose(1);">Program</a></li>
+            <li><a href="#speakers " onClick="menuClose(1);">Heros</a></li>
+            <li><a href="#sponsors" onClick="menuClose(1);">Sponsor</a></li>
+            
+
+
+          
+        </ul>
+
+    </header>
+
+    <div id="mobile_menu">
+        <div id="mobile_button" class="close_button" onClick="menuClose(1);">
+            <img src="images/mobil_close_menu.png" alt="close button" />
+        </div>
+        <ul>
+        <li><a href="./about.html " onClick="menuClose(1);">About</a></li>
+        <li><a href="#program " onClick="menuClose(1);">Program</a></li>
+        <li><a href="#speakers " onClick="menuClose(1);">Heros</a></li>
+        <li><a href="#sponsors" onClick="menuClose(1);">Sponsor</a></li>
+        </ul>
+
+        </div>
+      
+
+    <main>
+        <section id="cover" class="cover">
+        <h2><img src="./images/main_big.png" alt="title" /> </h2>
+
+            <div class="cover_extra">
+                This event aims to bring together business leaders and teams who want to learn and understand how
+                teamwork is managed
+                around the world for three days of exciting talks and impressive presentations.
+            </div>
+
+
+            <p>2022.02.18(SAT)- 20(MON) </p>
+            <p>Gran Salon Corferias, Bogotá-Colombia</p>
+
+
+
+        </section>
+        <section id="program" class="program">
+
+            <div>Main Program</div>
```

Comment:
- [ ] The middle section should have a column that includes a section header, divider, five individual articles, and a line of text at the bottom, please consider adding the header, separator and a line of text at the bottom

|Your design|CC design|
|---|---|
|![Screenshot_20220520_061041](https://user-images.githubusercontent.com/25227022/169455296-9b966b7d-9f3f-4c5f-929f-18fb2e442b21.png)|![Screenshot_20220520_061449](https://user-images.githubusercontent.com/25227022/169455365-49c3a0d8-add3-462f-9302-9573d7af87d8.png)|